react/basic-example/package.json

```diff
@@ -7,13 +7,19 @@
     "talkjs": "^0.3.0"
   },
   "devDependencies": {
-    "react-scripts": "1.0.13"
+    "react-scripts": "^2.1.8"
```

Comment:
not sure about the ^ - what if they ship a breaking change? wouldn't a fixed version be safer?

Reply:
`^` means compatible with version (which in-theory means there should be no breaking changes). This version works for the current example.

Reply:
yes, in theory.

i'm thinking that it might be good for all our examples to have hardcoded version numbers, because the JS ecosystem doesn't always take their own semver into high account. but i'm not too bullish on this one, your call.

Reply:
ah wait, i'm old fashioned. are we publishing `package-lock.json`? if not, we should add it for all examples and then the problem is solved, doesn't matter whether the package.json says ^ or not.

Reply:
Yes we are - some examples had `yarn.lock` files instead but I made sure they all are now `package-lock.json` instead.

Reply:
awesome! ok ignore my comment then :-)

(btw what's the purpose of "resolving" conversations, other than making it harder to find back?)